### PR TITLE
178 add proxy to service objects

### PR DESCRIPTION
**Pull request by VShutkina**
proxy support was added to service objects

### CODE REVIEW

**Comment by JDIReporter** [comment]
[550] -

**Comment by JDIReporter** [comment]
[551] -

**Comment by JDIReporter** [comment]
[551] - https://transfer.sh/Wp6At/jdi-dark-tests.tar.gz

**Comment by JDIReporter** [comment]
[551] - Allure report on Netlify: https://5e54cf0769501c0380a79683--jdi-dark.netlify.com

File: jdi-dark/src/main/java/com/epam/http/requests/RestMethod.java [hunk 1]
```diff
@@ -318,6 +319,20 @@ public void addFormParameters(FormParameter... params) {
...
+        //data.proxySpecifications.add(new ProxySpecification(proxyParams.host(), proxyParams.port(), proxyParams.scheme()));*/
+        //this.spec.proxy(proxyParams.host(), proxyParams.port(), proxyParams.scheme());
+        data.setProxySpecification(proxyParams.scheme(), proxyParams.host(), proxyParams.port());
+                /*.withScheme(proxyParams.scheme())
+                .and()
+                .withHost(proxyParams.host())
+                .and()
+                .withPort(proxyParams.port());*/
+    }
+
```
Comment (ElenaPoduzova) [inline jdi-dark/src/main/java/com/epam/http/requests/RestMethod.java]: Commented code - is it needed here?

Comment (VShutkina) [reply jdi-dark/src/main/java/com/epam/http/requests/RestMethod.java]: It's my mistake, commented code is removed

**Comment by JDIReporter** [comment]
[558] - https://transfer.sh/ANXB6/jdi-dark-tests.tar.gz

**Comment by JDIReporter** [comment]
[558] - Allure report on Netlify: https://5e54f5de13f8355cc68338e6--jdi-dark.netlify.com

**Pull request merged**